spec/expand/simple/lookup.spec

```diff
@@ -1,12 +1,31 @@
-%a
+%abc
 one
 two
 both
 
-%b:KEYS
+%def:KEYS
 CLUSTER
 TYPE
 
-%{b}:KEYS
+%{def}:KEYS
+CLUSTER
+TYPE
+
+%{a}bc
```

Comment:
This is interpreted as %{ {a}bc } and should error since there's no %abc cluster